docs/cli/use.rst

```diff
@@ -65,8 +65,22 @@ This creates a schema named ``view_data_collection_4_5_6``.
 
 If you need to summarize more than five collections, then you must :ref:`customize the schema's name<set-schema-name>`.
 
+.. _tables-only:
+
+Create persistent tables for all summary tables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, some summary tables are database `views <https://www.postgresql.org/docs/current/sql-createview.html>`__ and not persistent `tables <https://www.postgresql.org/docs/current/sql-createtable.html>`__, in order to save disk space.  Use the ``--tables-only`` option to make all summary tables into persistent tables.
+
+.. code-block:: bash
+
+    python ocdskingfisher-views-cli add-view 123 "The note" --name the_name --tables-only
+
+This option is useful if you want to create a schema that does not rely on data from the Kingfisher Process database.  This means that if you give users permissions to this schema only then they will have a useful subset of the data, without them needing to have permissions to all data within Kingfisher.
```

Comment:
```suggestion
Use this option if:

-  You want to allow a user to access the schema's tables, but not Kingfisher Process' tables
-  You want to make it easier for a user to discover the foreign key relationships between tables (for example, using ``\d <table>`` instead of ``\d+ <view>`` followed by ``\d <table>``)
-  You are :ref:`creating the Entity Relationship Diagram<create_erd>`
```